README.md

```diff
@@ -22,14 +22,83 @@ Note: Edge debugging via [Edge DevTools Protocol](https://docs.microsoft.com/en-
 * Any features that aren't script debugging.
 
 ## Getting Started
-We are working on creating and publishing a VS Code Extension for this project.
+For use inside VS Code:
+1. [Install the extension](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge)
+2. Restart VS Code and open the folder containing the project you want to work on.
 
 For use inside Visual Studio:
 1. Install the latest [Windows Insider Preview](https://insider.windows.com/en-us/getting-started/) build.
 2. Install the latest [Visual Studio 2017 Version 15.7 Preview](https://www.visualstudio.com/vs/preview/) build.
 3. Create an ASP.Net/ASP.Net Core Web Application.
```

Comment:
We should add a step - "Set a breakpoint in your JavaScript file" - this will prompt users to enable script debugging as its disabled by default starting with 15.7.